pylutron_caseta/smartbridge.py

```diff
@@ -141,21 +139,6 @@ def get_scene_by_id(self, scene_id):
         """
         return self.scenes[scene_id]
 
-    def get_value(self, device_id):
```

Comment:
This method didn't do what it claimed to do, and it was never used anywhere.  It published a read request to a zone status, but then returned the result of `self._writer.write(cmd)`, which is always `None`.  To correctly implement it, it would need to be async'ed with an awaited read statement to read the resulting read response with relevant zone status returned.

However, none of this is necessary, anyway, since all light level updates are posted to every open connection.  And it's not used or tested here or in home assistant.